function-images/tests/chained-function-serving/tracing/tracing.go

```diff
@@ -0,0 +1,80 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/exporters/trace/zipkin"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv"
+)
+
```

Comment:
refactor. create a type Tracer with various pointer receivers

[example](https://tour.golang.org/methods/4)

Reply:
decided to leave tracer as-is